leaderboard/service/get_top_percentage.go

```diff
@@ -0,0 +1,46 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	"github.com/topfreegames/podium/leaderboard/model"
+)
+
+const getTopPercentageServiceLabel = "get top percentage"
+
+// GetTopPercentage retrieves top x% members from the leaderboard.
+func (s *Service) GetTopPercentage(ctx context.Context, leaderboardID string, pageSize, amount, maxMembers int, order string) ([]*model.Member, error) {
+	if amount < 1 || amount > 100 {
+		return nil, fmt.Errorf("Percentage must be a valid integer between 1 and 100.")
+	}
+
+	if order != "desc" && order != "asc" {
+		order = "desc"
+	}
```

Comment:
I am stopping to enforce wrong orders to `desc` in the new version of `leaderboard`, once that is better ask the client to send the correct argument. Could you remove this?

Reply:
Or better throw a new Error